src/BlazorWebView/src/Maui/iOS/BlazorWebViewHandler.iOS.cs

```diff
@@ -95,10 +95,12 @@ protected override WKWebView CreatePlatformView()
 				AutosizesSubviews = true
 			};
 
-#if MACCATALYST13_3_OR_GREATER || IOS16_4_OR_GREATER
-			// Enable Developer Extras for Catalyst/iOS builds for 16.4+
-			webview.SetValueForKey(NSObject.FromObject(DeveloperTools.Enabled), new NSString("inspectable"));
-#endif
+			if (UIDevice.CurrentDevice.CheckSystemVersion(16, 4))
```

Comment:
Something like this might be. better:

https://github.com/dotnet/maui/blob/087993f09d790658587861308194831a169ef447/src/Core/src/Platform/iOS/ContentView.cs#L16

Reply:
I had no idea that existed, thanks for pointing that out!

Updated with https://github.com/dotnet/maui/pull/16631/commits/f109c5fe497b301dec32dfcd6fb0ce83397de240, checking it on iOS and Catalyst and it passed through correctly and set it.